src/features/scheduled-messages.ts

```diff
@@ -21,13 +39,77 @@ export const MESSAGE_SCHEDULE: MessageConfig[] = [
     }
   }
   */
-];
+  {
+    interval: FREQUENCY.daily,
+    postTo: [{ channelId: CHANNELS.jobBoard }],
```

Comment:
Would it be worth combining these two properties so you could do something like this? We could have the same message on different intervals for different channels.

```js
postTo: [
   { channelId, interval },
]```

Reply:
Oh I'm into that

Reply:
Pushed